collections/_article/godotsharp-packages-net8.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Godot C# packages move to .NET 8"
+excerpt: "On moving to the latest .NET LTS release, and what it means for users."
+categories: ["progress-report"]
+author: Raul Santos
+image: /storage/blog/covers/progress-update-csharp-3.webp
```

Comment:
TODO: Add cover image. This webp doesn't exist.

Reply:
Would something like this work? 
Also, for this one I would like us to try with a jpg, trying to make sure the reason why our images don't show in Bluesky is not because of webp
![dot-net-update](https://github.com/user-attachments/assets/d6ac7778-0ea9-46cd-94be-e950751a26c0)